bindings/postgres/postgres.go

```diff
@@ -212,6 +242,62 @@ func (p *Postgres) exec(ctx context.Context, sql string, args ...any) (result in
 	return res.RowsAffected(), nil
 }
 
+func (p *Postgres) fetchOne(ctx context.Context, sql string, args ...any) (result []byte, err error) {
+
+	rows, err := p.db.Query(ctx, sql, args...)
+	if err != nil {
+		return nil, fmt.Errorf("error executing query: %w", err)
+	}
+
+	rs := make(map[string]any, 0)
+	for rows.Next() {
+		val, rowErr := rows.Values()
+		if rowErr != nil {
+			return nil, fmt.Errorf("error reading result '%v': %w", rows.Err(), rowErr)
```

Comment:
`rows.Err()` and `rowErr` should be identical per docs

```suggestion
			return nil, fmt.Errorf("error reading row: %w", rowErr)
```